src/content/docs/style-guide/processes-procedures/rename-or-redirect-document.mdx

```diff
@@ -18,12 +18,12 @@ This document describes how to change the title of a document and how to create,
 
 ## URL format [#title-url]
 
-Changing a document's title also affects its URL. The URL is the text of the title, where hyphens `-` replace spaces. Drupal automatically removes ["short words"](/admin/config/search/path/settings) (`the`, `a`, `for`, `via`, etc.) from URL paths.
+Changing a document's title also affects its URL. The URL is the text of the title, where hyphens `-` replace spaces.
```

Comment:
This is no longer true in the new Gatsby site. The URL is determined by the filepath in `src/content`. So a file at `src/content/docs/test.mdx` will create a page at `/docs/test`. The `title` is now just to show on the top of the page, and will render as part of the title in the tab of the browser.

Reply:
I've rewritten to reflect the reality here.